packages/localization/src/Deployer.js

```diff
@@ -121,6 +136,64 @@ const createTranslation = async (keyId, translationString) => {
   return _fetch(`/translations`, formData, HttpMethod.POST);
 };
 
+const deleteKey = async (keyId, keyName) => {
+  if (keyId != null && keyId !== '') {
+    _log(`Deleting key ${keyName} with id ${keyId}`);
+    return _fetch(`/keys/${keyId}`, null, HttpMethod.DELETE);
+  }
+};
+
+// finding unused keys inside our codebase
+
+const findFiles = async (key: string) => {
+  const app = await findInFiles.findSync(key, 'app');
+  const accessibility = await findInFiles.find(key, 'packages/accessibility');
+  const assets = await findInFiles.find(key, 'packages/assets');
+  const config = await findInFiles.find(key, 'packages/config');
+  const navigation = await findInFiles.find(key, 'packages/navigation');
+  const orbit = await findInFiles.find(key, 'packages/orbit');
+  const pdf = await findInFiles.find(key, 'packages/pdf');
+  const playground = await findInFiles.find(key, 'packages/playground');
+  const relay = await findInFiles.find(key, 'packages/relay');
+  const rnmodules = await findInFiles.find(key, 'packages/rnmodules');
+  const shared = await findInFiles.find(key, 'packages/shared');
```

Comment:
Also maybe add folders to an array, and loop that array, I guess it scales better.

Reply:
That's what I did first, but then the performance was much worse somehow. It took twice the amount of time.

Reply:
Ok, then please add a comment as of why you chose this approach 😊